sympy/physics/vector/frame.py

```diff
@@ -873,40 +871,26 @@ def orient_body_fixed(self, parent, angles, rotation_order):
         if rot_order not in approved_orders:
             raise TypeError('The rotation order is not a valid order.')
 
-        parent_orient_body = []
         if not (len(amounts) == 3 & len(rot_order) == 3):
             raise TypeError('Body orientation takes 3 values & 3 orders')
-        a1 = int(rot_order[0])
-        a2 = int(rot_order[1])
-        a3 = int(rot_order[2])
-        parent_orient_body = (self._rot(a1, amounts[0]) *
-                              self._rot(a2, amounts[1]) *
-                              self._rot(a3, amounts[2]))
-
-        self._dcm(parent, parent_orient_body)
-
-        try:
-            from sympy.polys.polyerrors import CoercionFailed
-            from sympy.physics.vector.functions import kinematic_equations
-            q1, q2, q3 = amounts
-            u1, u2, u3 = symbols('u1, u2, u3', cls=Dummy)
-            templist = kinematic_equations([u1, u2, u3], [q1, q2, q3],
-                                           'body', rot_order)
-            templist = [expand(i) for i in templist]
-            td = solve(templist, [u1, u2, u3])
-            u1 = expand(td[u1])
-            u2 = expand(td[u2])
-            u3 = expand(td[u3])
-            wvec = u1 * self.x + u2 * self.y + u3 * self.z
-            # NOTE : SymPy 1.7 removed the call to simplify() that occured
-            # inside the solve() function, so this restores the pre-1.7
-            # behavior. See:
-            # https://github.com/sympy/sympy/issues/23140
-            # and
-            # https://github.com/sympy/sympy/issues/23130
-            wvec = wvec.simplify()
-        except (CoercionFailed, AssertionError):
-            wvec = self._w_diff_dcm(parent)
+        rot_matrices = (self._rot(int(rot_order[0]), amounts[0]),
+                        self._rot(int(rot_order[1]), amounts[1]),
+                        self._rot(int(rot_order[2]), amounts[2]))
+        self._dcm(parent, rot_matrices[0] * rot_matrices[1] * rot_matrices[2])
+
+        from sympy.physics.vector.functions import dynamicsymbols
+
+        rot_vecs = [zeros(3, 1) for _ in range(3)]
+        for i, order in enumerate(rot_order):
+            rot_vecs[i][int(order) - 1] = amounts[i].diff(dynamicsymbols._t)
+        # Express in parent frame
+        # u1, u2, u3 = rot_vecs[0] + rot_matrices[0] * (
+        #     rot_vecs[1] + rot_matrices[1] * rot_vecs[2])
+        # wvec = u1 * parent.x + u2 * parent.y + u3 * parent.z
```

Comment:
Has this been left in by accident?

Reply:
See [this comment](https://github.com/sympy/sympy/pull/24085#issuecomment-1257951598)